File: wireless/commands/list-example-1/list-example-1.5.x.php
<?php
// Download the **Next-Gen** twilio-php library from:
// twilio.com/docs/libraries/php#installation-nextgen
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Your Account SID from www.twilio.com/console
$sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
// Your Auth Token from www.twilio.com/console
$token = "your_auth_token";

// Initialize Twilio Rest Client
$client = new Client($sid, $token);

// List all commands
$commands = $client->preview->wireless->commands->stream();

foreach ($commands as $cmd) {
    echo $cmd->command . PHP_EOL;
}

File: wireless/sims-data-session/list-example-1/list-example-1.5.x.cs
// Download the twilio-csharp library from twilio.com/docs/libraries/csharp
using System;
using Twilio;
using Twilio.Rest.Wireless.V1.Sim;

class Example
{
    static void Main(string[] args)
    {
        // Find your Account SID and Auth Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var dataSessions = DataSessionResource.Read("DEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA");

        foreach (var dataSession in dataSessions)
        {
            Console.WriteLine(dataSession);
        }
    }
}
